examples/nixnet/can-signal-single-point-output.py

```diff
@@ -0,0 +1,125 @@
+r""" Write Signal Data.
```

Comment:
There seems to be an empty `PortExamplesC-Python` file above. I couldn't add a comment there, so adding it here. Make sure remove that file before you complete the PR.